src/root.css

```diff
@@ -0,0 +1,9 @@
+:root {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
margin to `body` tag
font-family to `html` tag

Reply:
done